db/migrate/20150122235703_create_admins.rb

```diff
@@ -1,12 +0,0 @@
-class CreateAdmins < ActiveRecord::Migration
-  def change
-    create_table :admins do |t|
-      t.string :first_name
-      t.string :last_name
-      t.string :email
-      t.string :password_digest
-
-      t.timestamps null: false
-    end
-  end
-end
```

Comment:
This breaks everyones db. Removing migrations after everyone else has them and they have been run causes `rake db:migrate` to break.